command/aarelayer/sendtx/sendtx.go

```diff
@@ -0,0 +1,167 @@
+package sendtx
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/0xPolygon/polygon-edge/command/aarelayer/service"
+	"github.com/0xPolygon/polygon-edge/command/polybftsecrets"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/wallet"
+	"github.com/0xPolygon/polygon-edge/helper/common"
+	"github.com/spf13/cobra"
+	"github.com/umbracle/ethgo"
+)
+
+const (
+	waitForReceiptTime    = time.Second * 10
+	numWaitReceiptRetries = 50
+)
+
+var params aarelayerSendTxParams
+
+func GetCommand() *cobra.Command {
+	configCmd := &cobra.Command{
+		Use:     "sendtx",
+		Short:   "sends account abstraction transaction to the relayer",
+		PreRunE: runPreRun,
+		RunE:    runCommand,
+	}
+
+	setFlags(configCmd)
+
+	return configCmd
+}
+
+func runPreRun(_ *cobra.Command, _ []string) error {
+	return params.validateFlags()
+}
+
+func runCommand(cmd *cobra.Command, _ []string) error {
+	cmd.SilenceUsage = true
+	cmd.SilenceErrors = true
```

Comment:
What this line actually does? Does it suppress all the errors printing?

Reply:
No, it just prevents to show usage whenever errors occurs and prevent duplicate output like this
```
Error: failed to read account data: unable to read secret from disk (/home/igor/polygon-keeeeey/consensus/validator.key), open /home/igor/polygon-keeeeey/consensus/validator.key: no such file or directory
failed to read account data: unable to read secret from disk (/home/igor/polygon-keeeeey/consensus/validator.key), open /home/igor/polygon-keeeeey/consensus/validator.key: no such file or directory
```